google-http-client/src/main/java/com/google/api/client/http/javanet/NetHttpTransport.java

```diff
@@ -60,6 +59,12 @@
  * @author Yaniv Inbar
  */
 public final class NetHttpTransport extends HttpTransport {
+  private static Proxy defaultProxy() {
+    return new Proxy(
+        Proxy.Type.HTTP, new InetSocketAddress(
+          System.getProperties().getProperty("https.proxyHost"),
```

Comment:
Let's change this to just System.getProperty() to match the rest of the project.

Reply:
Updated the Pull Request to reflect it.